File: gaming/src/main/java/com/oruga/gaming/dto/UsuarioDto.java
package com.oruga.gaming.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class UsuarioDto {
    private Long id;
    private String email;
    private String nombreUsuario;
    private Boolean esModerador;
    private Boolean estaBaneado;
    private String urlFotoPerfil;
    private String descripcion;
    private Integer cantidadPublicaciones;
    private Integer cantidadMensajes;
}

File: MyApplication3/app/src/main/java/com/example/myapplication/data/remote/dto/UsuarioDTO.kt
package com.example.myapplication.data.remote.dto

data class UsuarioResponse(
    val id: Long,
    val email: String,
    val nombreUsuario: String,
    val esModerador: Boolean,
    val estaBaneado: Boolean,
    val fechaBaneo: String?,
    val razonBaneo: String,
    val urlFotoPerfil: String,
    val descripcion: String,
    val steamId: String,
    val discordId: String,
    val fechaCreacion: String,
    val ultimaConexion: String,
    val estadoConexion: Boolean,
    val cantidadPublicaciones: Int,
    val cantidadMensajes: Int,
    val cantidadReportes: Int
)

data class ActualizarPerfilRequest(
    val descripcion: String?,
    val urlFotoPerfil: String?
)